lib/cPanel/PublicAPI.pm

```diff
@@ -93,6 +94,13 @@ sub new {
         $self->{'ip'} = '127.0.0.1';
     }
 
+    $self->{'ua'} = HTTP::Tiny->new(
+        agent      => "cPanel::PublicAPI/$VERSION ",
+        verify_SSL => $self->{'ssl_verify_mode'},
+        keep_alive => $self->{'keepalive'},
```

Comment:
Minor: This is now the only part of the code that uses `$self->{'keepalive'}`.  We may want to make it a local variable.